src/lib/getMajorVersion.js

```diff
@@ -0,0 +1,10 @@
+/**
+ *
+ * @param {string} version
+ * @returns {number} The major version of ReactDOM
```

Comment:
Since this function might be used with other packages or strings, maybe we drop the bit about 'ReactDOM'? An example might help too:

`@example getMajorVersion('17.0.2') // 17`